### PR TITLE
Requiem 3.0.0

### PR DESCRIPTION
**Pull request by ProbablyManuel**
FYI: `Requiem - Legendary Bugsmasher Edition.esp` has been merged into `Requiem.esp` in version 3.0.0. This solves the load order problems related to the bugsmasher.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -9415,10 +9415,10 @@ plugins:
       - <<: *doNotClean
         condition: 'version("Requiem.esp", "1.9.3", >=)'
       - <<: *patchIncluded
-        condition: 'active("Unofficial Skyrim Patch.esp") and not active("Requiem - Bugsmasher Edition.esp")'
+        condition: 'version("Requiem.esp", "2.0.0", <) and active("Unofficial Skyrim Patch.esp") and not active("Requiem - Bugsmasher Edition.esp")'
```
Comment (Ortham) [inline masterlist.yaml:9418]: Should this version number not be `3.0.0`?

Comment (MacSplody) [reply masterlist.yaml:9418]: Support for `Unofficial Skyrim Patch.esp` was dropped in 2.0.0.
So I think the condition is correct.
https://requiem.atlassian.net/wiki/spaces/RD/pages/403439801/Requiem+2.0.0+-+The+Phantom+Agony

> With USKP ousted we're now only usable with USLEEP, so dependent on all DLCs (though with Dragonborn disabled).

Comment (Ortham) [reply masterlist.yaml:9418]: Oh, right.

Comment (ProbablyManuel) [reply masterlist.yaml:9418]: Yes, `Requiem - Bugsmasher Edition.esp` was removed in 2.0.0.

**Review by MacSplody — APPROVED** [review]
@Kaenguru-Knobel-Kette @Ortham Looks fine to me.

**Pull request merged**